glue/core/data.py

```diff
@@ -1179,7 +1179,8 @@ def delay_callbacks():
                     label = axis_label(self.coords, i)
                     cid = self.add_component(comp, label)
                     self._world_component_ids.append(cid)
-                self._set_up_coordinate_component_links(ndim)
+                if settings.AUTO_COMPUTE_COORDS_LINKS:
+                    self._set_up_coordinate_component_links(ndim)
```

Comment:
I assume the _creation_ of the components above is still required.

Reply:
We could remove that but it would be a bigger change - best to try this in a separate PR